setup.py

```diff
@@ -18,6 +17,8 @@
 # Concatenate all the values together for 'all'
 extras["all"] = list(chain.from_iterable(ex_extras.values()))
 
+# packages = find_packages() + ['plasmapy.addons', ]
```

Comment:
An extra commented out line?

Reply:
Ha! I originally thought you were suggesting this and I was beyond confused.  This was left over from my playing around, it should not be here.